doc/sphinx/Makefile.sphinx

```diff
@@ -63,6 +63,13 @@ html: symlink-docs check-sphinxbuild clean-build
 	@echo
 	@echo "Build finished. The HTML pages are in "'$(BUILDPATH)'"/html."
 
+check: symlink-docs check-sphinxbuild clean-build
```

Comment:
`make check` only generates a `build/html/output.txt` with the errors listed, so I don't think there's any benefit to _not_ cleaning for each build.